.github/CONTRIBUTING.md

```diff
@@ -1,34 +1,90 @@
 # Contributing to Kitura
 
-We welcome contributions, and request you follow these guidelines.
+This document contains information and guidelines about making contributions. For help, contact us on [Slack](http://swift-at-ibm-slack.mybluemix.net/).
 
- - [Raising issues](#raising-issues)
+We love contributions to the Kitura project, but request you follow the guidelines below.
+
+ - [Getting Started](#getting-started)
+ - [Pull Requests](#pull-requests)
+ - [Coding Style](#coding-style)
  - [Contributor License Agreement](#contributor-license-agreement)
- - [Coding Standards](#coding-standards)
+ - [Asking Questions](#asking-questions)
+ - [Reporting issues](#reporting-issues)
+ - [Additional Resources](#additional-resources)
+
+
+## Getting Started
+
+Please ensure you follow [the Kitura coding standards](https://github.com/IBM-Swift/Kitura/blob/master/Documentation/CodeConventions.md). This keeps the source code tidy, and allows [Swift Lint]((https://github.com/realm/SwiftLint)) to work with the project.
```

Comment:
The link-through text doesn't need the word `the` in it, it should read `Kitura coding standards` just make `the` part of the sentence.
The Swift Lint link here doesn't work either - perhaps it's the extra brackets?
Please can you also drop the , before the `and` in the second sentence (to be honest this could be one sentence if you like as the `. This` could be `, this`

Reply:
Can you change this to `Please follow the Kitura coding standards` I don't think we need the `ensure you` wording as the PR wouldn't get past a reviewer anyway (or shouldn't!) if it didn't.

Reply:
✅